cppcache/include/geode/Exception.hpp

```diff
@@ -52,9 +52,9 @@ class CPPCACHE_EXPORT Exception : public std::exception {
   /** Creates an exception.
    * @param  msg1 message pointer, this is copied into the exception.
    **/
-  Exception(const char* msg1);
+  explicit Exception(const char* msg1);
```

Comment:
I think it is only really necessary when the first arg is a ref to avoid a conversation copy construct. Though it probably doesn't hurt here.